src/workbench/SideBarHeader.tsx

```diff
@@ -37,6 +37,27 @@ const SideBarHeader = () => {
   const [query, setQuery] = useState("");
   const [results, setResults] = useState<SearchResults | undefined>();
   const isUnmounted = useIsUnmounted();
+
+  // When we add more keyboard shortcuts, we should pull this up and have a CM-like model of the
+  // available actions and their shortcuts, with a hook used here to register a handler for the action.
+  useEffect(() => {
+    const isMac = /Mac/.test(navigator.platform);
+    const keydown = (e: KeyboardEvent) => {
+      if (
+        e.key === "f" &&
```

Comment:
```suggestion
        (e.key === "F" || e.key === "f") &&
```
Handles default case (capital letter via `shift` key), and lower case if `Caps Lock` is on.

Reply:
I can't even test this as I remapped Caps Lock to Escape long ago!